### PR TITLE
allow using pull-drain-gently in migrate

### PR DESCRIPTION
**Pull request by staltz**
For issue #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,6 +141,10 @@ const sbot = SecretStack({appKey: caps.shs})
 
 However, note that while the old log exists, it will be continously migrated to the new log, and ssb-db2 forbids you to use its database-writing APIs such as `add()`, `publish()`, `del()` and so forth, to prevent the two logs from diverging into inconsistent states. The old log will remain the source of truth and keep getting copied into the new log, until the old log file does not exist anymore.
 
+### Migrating gently
+
+If migrating to the new log is not a priority over other computations and UI responses, you can choose to do the migration tasks gently, respecting current CPU usage. To enable this mode, set `config.db2.migrateMaxCpu` to a number between 1 and 100 representing the maximum CPU load allowed for this task. Underneath, it will use [`pull-drain-gently`](https://gitlab.com/staltz/pull-drain-gently/). You can also set `config.db2.migrateMaxPause`.
```
Comment (arj03) [inline README.md:146]: minor thing, it would be good to define what migrateMaxPause is.

Comment (staltz) [reply README.md:146]: Oh yeah, it's a pull-drain-gently option, I could point the reader to that.

**Comment by arj03** [comment]
Looks good, would merge except there merge conflict from the other PR

**Comment by staltz** [comment]
I'll fix the conflict tomorrow. Should be easy

**Comment by staltz** [comment]
Conflicts resolved :heavy_check_mark:

**Comment by arj03** [comment]
Looks good

**Pull request merged**